src/main/scala/loquats/phylogeneticTree.scala

```diff
@@ -59,7 +59,7 @@ case object visualizations {
         val client = RClient(serializeOutput = true)
         val inputPath = productiveClonotypesTSV.getCanonicalPath()
         val outputPath = output(data.viz.phylogeneticTree).getCanonicalPath()
-        client eval s"""
+        client eval raw"""
```

Comment:
@agarciamontoro I just remembered about `raw` interpolator, with it you don't need to double escape things:

```scala
@ raw"foo \n \\n ${1+1} $${1+1}"
res0: String = "foo \\n \\\\n 2 ${1+1}"
@ println(raw"foo \n \\n ${1+1} $${1+1}")
foo \n \\n 2 ${1+1}
```

Reply:
Oh, this is great, thanks for the tip!